src/vmm/src/devices/virtio/mmio.rs

```diff
@@ -239,7 +245,13 @@ impl MmioTransport {
                     }
                     0x34 => self.with_queue(0, |q| u32::from(q.get_max_size())),
                     0x44 => self.with_queue(0, |q| u32::from(q.ready)),
-                    0x60 => self.interrupt_status.load(Ordering::SeqCst),
+                    0x60 => {
+                        if !self.is_vhost_user {
+                            self.interrupt_status.load(Ordering::SeqCst)
+                        } else {
+                            VIRTIO_MMIO_INT_VRING
```

Comment:
let's add a comment on why we don't need to care about the bit 1 here.

Reply:
Wanted to comment this as well. I guess the assumption here is that we will never need to update the config space. Is that right?

Reply:
yes